pages/mesosphere/dcos/services/nifi/1.0.0-1.9.2/release-notes/index.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes
+title: Release Notes
+menuWeight: 5
+excerpt: Discover the new features, updates, and known limitations in this release of the NiFi Service
+featureMaturity:
+enterprise: false
+model: /mesosphere/dcos/services/nifi/data.yml
+render: mustache
+---
+
+# Release Notes for NiFi Service version 1.0.0-1.9.2
+
+## Bug Fix
+- Fix removing nodes from cluster after upgrade/restart.
```

Comment:
This page is going to be important :)

Reply:
I think we also want to mention that:
- the UID for `nobody` is now Debian default rather than `99`.
- it's based on more recent SDK which supports quota enforcement on 2.0

Reply:
I have added @porridge's comments to the Release Notes.

Reply:
I have added Marcin's comments to the release notes.